back/config/routes.yaml

```diff
@@ -3,7 +3,14 @@
 #    controller: App\Controller\DefaultController::index
 
 post_create:
-  path: /posts
+  path: /create-post
   controller:
-    App\Infrastructure\Controller\PostController::create
+    App\Infrastructure\Controller\CreatePostController::create
   methods: [POST]
+
+get-posts:
```

Comment:
Not really important but naming is inconsistent here `post_create` vs `get-posts`